frame/contracts/src/tests.rs

```diff
@@ -1630,7 +1630,7 @@ fn cannot_self_destruct_in_constructor() {
 #[test]
 fn check_block_gas_limit_works() {
 	ExtBuilder::default().block_gas_limit(50).build().execute_with(|| {
-		let info = DispatchInfo { weight: 100, class: DispatchClass::Normal, pays_fee: true };
+		let info = DispatchInfo { weight: 100, class: DispatchClass::Normal, pays_fee: Pays::Yes };
```

Comment:
So except for the high level `bin/node/executor` tests, I will keep this assertions explicit intentionally, instead of doing `Default::default()` which is unclear and flaky.